src/applications/model/crazytaxi-streaming-server.cc

```diff
@@ -0,0 +1,123 @@
+    Ptr<ConstantRandomVariable> iat_cbr_audio = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue(10));
+    AddNewTraffic (pkt_cbr_audio, iat_cbr_audio);
+
+    /** Add Cursor stream */
+    Ptr<ConstantRandomVariable> pkt_cursor = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue(28));
+    Ptr<ConstantRandomVariable> iat_cursor = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue(50));
+    AddNewTraffic (pkt_cursor, iat_cursor);
+
+    /** Add VBR audio stream */
+    // Packet size
+    Ptr<ConstantRandomVariable> pkt_vbr_audio1 = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue(244));
+    Ptr<ConstantRandomVariable> pkt_vbr_audio2 = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue(1384));
+
+    Ptr<GamingTrafficModel> pkt_vbr_audio = CreateObject<GamingTrafficModel> ();
+    pkt_vbr_audio->SetRngs ({pkt_vbr_audio1, pkt_vbr_audio2});
+    pkt_vbr_audio->SetProb ({0.0776, 0.9224});
+
+    // Packet inter-arrival time
+    Ptr<ConstantRandomVariable> iat_vbr_audio = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue(50));
+    AddNewTraffic (pkt_vbr_audio, iat_vbr_audio);
+
+    /** Add video stream */
+    // Packet size
+    Ptr<UniformRandomVariable> pkt_video1 = CreateObjectWithAttributes<UniformRandomVariable> ("Min", DoubleValue(0),
+                                                                                              "Max", DoubleValue(1355));
+    Ptr<ConstantRandomVariable> pkt_video2 = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue(1356));
+
+    Ptr<GamingTrafficModel> pkt_video = CreateObject<GamingTrafficModel> ();
+    pkt_video->SetRngs ({pkt_video1, pkt_video2});
+    pkt_video->SetProb ({0.3606, 0.6394});
+
+    // Packet inter-arrival time
+    Ptr<ConstantRandomVariable> iat_video1 = CreateObjectWithAttributes<ConstantRandomVariable> ("Constant", DoubleValue(0));
+    Ptr<LogNormalRandomVariable> iat_video2 = CreateObjectWithAttributes<LogNormalRandomVariable> ("Mu", DoubleValue(0.34),
+                                                                                                  "Sigma", DoubleValue(1.73));
+
+    Ptr<GamingTrafficModel> iat_video = CreateObject<GamingTrafficModel> ();
+    iat_video->SetRngs ({iat_video1, iat_video2});
+    iat_video->SetProb ({0.5725, 0.4275});
+
+    AddNewTraffic (pkt_video, iat_video);
+  }
+
+  void
+  CrazyTaxiStreamingServer::AddNewTraffic (Ptr<RandomVariableStream> pkt, Ptr<RandomVariableStream> iat)
+  {
+    NS_LOG_FUNCTION (this);
+    m_pksList.push_back (pkt);
+    m_iatList.push_back (iat);
+  }
+
+  void
+  CrazyTaxiStreamingServer::Send (void)
+  {
+    NS_LOG_FUNCTION (this);
+    for (unsigned i = 0; i < m_pksList.size(); ++i)
+    {
+      SendPacket (m_pksList[i], m_iatList[i]);
+    }
```

Comment:
this could be defined in GamingStreamingServer instead since it's in common for every child class